src/runtime_src/core/common/system.cpp

```diff
@@ -69,4 +69,22 @@ bdf2index(const std::string& bdfStr)
   return instance().bdf2index(bdfStr);
 }
 
+void
+setup_terminal()
+{
+  instance().setup_terminal();
+}
+
+void
+restore_terminal()
+{
+  instance().restore_terminal();
+}
```

Comment:
How are these APIs used?  What happens in setup is called without restore?   
I ask out of concern for exception safety.

Reply:
Ansi Escape Sequences are supported on Unix like systems by default but Windows does not have that support. From Windows 10 Version 1511 this support is provided but is not enabled by default, Our program should do that. Here setup_terminal function does that and when you call restore_terminal  Ansi Escape chars are not interpreted from that point.

If setup is called without restore terminal still can interpret Ansi Escape Chars if provided in the program.

Reply:
Please let me know how the API is used.  If the code is 
```
setup();
dosomething();
restore();
```
Then it is not valid code and must be changed.